Package.swift

```diff
@@ -8,7 +8,7 @@ let package = Package(
     name: "mlx-swift",
 
     platforms: [
-        .macOS(.v13),
+        .macOS("13.3"),
```

Comment:
Should this not be `.v13_3`?

Reply:
there's no .v13_3 enum case afaik
![Screenshot 2024-02-22 at 15 29 52](https://github.com/ml-explore/mlx-swift/assets/758033/33174b04-1ff7-41e8-aba2-af1f86676c2c)